cmake/ace_group.cmake

```diff
@@ -63,7 +63,7 @@ function(_opendds_vs_force_static)
   endif()
 endfunction()
 
-if(MSVC AND OPENDDS_STATIC)
+if(MSVC AND OPENDDS_STATIC AND NOT ${OPENDDS_STATIC_RUNTIME} STREQUAL "OFF")
```

Comment:
```suggestion
if(MSVC AND OPENDDS_STATIC AND NOT OPENDDS_STATIC_RUNTIME)
```

Reply:
If the user doesn't specify the option to the configure script, the variable will be the empty string instead of OFF or ON.  CMake would treat the empty string as a false value.

Reply:
What about:

```suggestion
if(MSVC AND OPENDDS_STATIC AND NOT (OPENDDS_STATIC_RUNTIME STREQUAL "" OR  OPENDDS_STATIC_RUNTIME))
```

I think it'd be better if it's not dependent on a specific false value.

Reply:
Actually this still seems wrong...

Reply:
I just realized I inverted `OPENDDS_STATIC_RUNTIME` when I originally commented.

Reply:
Maybe we should only set `OPENDDS_STATIC_RUNTIME` when `--static-runtime` or `--no-static-runtime` is passed so it's only ever true, false, or undefined. Then this could be:

```suggestion
if(MSVC)
  if(OPENDDS_STATIC AND NOT DEFINED OPENDDS_STATIC_RUNTIME)
    set(OPENDDS_STATIC_RUNTIME TRUE)
  endif()
  if(OPENDDS_STATIC_RUNTIME)
```